animareflection/ui
# Minimize Library Entry Points with Client Component Focus

## Description

##### Task link: https://trello.com/c/zmRNXJwc/224-refactor-library-structure-with-client-component-focus

Update library structure to minimize entry points by putting focus on client components. New entry points are as follows:

1) `index.ts` (main entry point for root cascade layers, backfills, preset, components, and hooks).
2) `index.next.ts` (entry point for NextJS optimized components)

## Test Steps

1) Verify ALL component and recipe structures seem sound (emphasis on new recipe pattern to utilize `@ark-ui/anatomy`).
2) Verify that components render appropriately in storybook and downstream example apps.
3) verify that all tests pass.
